src/main.rs

```diff
@@ -73,12 +73,20 @@ fn run() -> Result<(), MainError> {
         eprintln!("> {command}\n");
         run_script(name, command, cwd)
     };
+    let command_string = |name: &str, args: Vec<String>| {
+        let mut commands = Vec::with_capacity(args.len() + 1);
+        commands.push(name.to_string());
+        let mut args = args;
+        commands.append(&mut args);
+        commands.join(" ")
+    };
```

Comment:
Is it really necessary to allocate a whole `Vec` just to join a string?

```suggestion
    let command_string = |name: &str, args: &[String]| {
        let mut string = name.to_string();
        for arg in args {
            string += " ";
            string += arg;
        }
        string
    };
```